contracts/interfaces/V3SpokePoolInterface.sol

```diff
@@ -222,7 +222,6 @@ interface V3SpokePoolInterface {
     error DisabledRoute();
     error InvalidQuoteTimestamp();
     error InvalidFillDeadline();
-    error InvalidExclusiveRelayer();
```

Comment:
Driveby change - this error is unused. It was dropped with the recent changes to permit relative exclusivity.